Dockerfile-local

```diff
@@ -10,19 +10,37 @@ ARG DEBIAN_FRONTEND=noninteractive
 
 ENV DJANGO_PRODUCTION=False
 
+# Install custom Pango version
+
+# Install FreeType2 then HarfBuzz then reinstall FreeType2
+RUN apt-get update && apt-get install -y \
+  bzip2 \
+  gperf \
+  libbz2-dev \
+  libffi-dev \
+  libtinfo-dev \
+  libmount-dev \
+  libicu52 \
+  libpcre3 \
+  libglib2.0-dev \
+  libgdk-pixbuf2.0-dev \
+  libcairo2-dev
+
+RUN mkdir /pango-install/
+COPY ./csunplugged/pango-install.sh /pango-install/
+RUN chmod +x /pango-install/pango-install.sh
+RUN /pango-install/pango-install.sh
+
 # Install packages (including Weasyprint dependencies), running of Python 3.4.2
 RUN apt-get update && apt-get install -y \
       python3 \
       python3-dev \
       python3-pip \
-      libffi-dev \
-      libcairo2-dev \
       python-dev \
       python-pip \
       python-lxml \
       python-cffi \
-      libpango1.0-0 \
-      libgdk-pixbuf2.0-0 \
+      fontconfig \
```

Comment:
Which part are you referring to? I added the `fontconfig` for the fonts, but I can't remember anymore if it's still required (pretty sure it is).